src/lib/job.rs

```diff
@@ -40,25 +41,39 @@ impl JobDesc {
             pr_num: pr_num,
             head_ref: String::from(head_ref),
             status: JobStatus::Created,
+            pid: 0,
         }
     }
 
     pub fn build_log_url(&self) -> String {
-        format!("{}/{}", config::BUILD_LOG_BASE_URL, &self.id)
+        format!("{}/{}", *config::BUILD_LOG_BASE_URL, &self.id)
```

Comment:
Is the *required because of `lazy_static`?

Reply:
Yes